header.go

```diff
@@ -382,6 +385,48 @@ func (h *RequestHeader) SetMultipartFormBoundaryBytes(boundary []byte) {
 	h.SetContentTypeBytes(h.bufKV.value)
 }
 
+// SetTrailer sets header Trailer value chunked response.
+func (h *ResponseHeader) SetTrailer(trailer string) {
+	for i := -1; i+1 < len(trailer); {
+		trailer = trailer[i+1:]
+		i = strings.IndexByte(trailer, ',')
+		if i < 0 {
+			i = len(trailer)
+		}
+		key := trailer[:i]
+		for len(key) > 0 && key[0] == ' ' {
+			key = key[1:]
+		}
+		for len(key) > 0 && key[len(key)-1] == ' ' {
+			key = key[:len(key)-1]
+		}
+		h.bufKV.key = append(h.bufKV.key[:0], key...)
+		normalizeHeaderKey(h.bufKV.key, h.disableNormalizing)
+		h.trailer = setArgBytes(h.trailer, h.bufKV.key, nil, argsNoValue)
+	}
+}
+
+// SetTrailerBytes sets Trailer header value for chunked response.
+func (h *ResponseHeader) SetTrailerBytes(trailer []byte) {
+	for i := -1; i+1 < len(trailer); {
+		trailer = trailer[i+1:]
+		i = bytes.IndexByte(trailer, ',')
+		if i < 0 {
+			i = len(trailer)
+		}
+		key := trailer[:i]
+		for len(key) > 0 && key[0] == ' ' {
+			key = key[1:]
+		}
+		for len(key) > 0 && key[len(key)-1] == ' ' {
+			key = key[:len(key)-1]
+		}
+		h.bufKV.key = append(h.bufKV.key[:0], key...)
+		normalizeHeaderKey(h.bufKV.key, h.disableNormalizing)
+		h.trailer = setArgBytes(h.trailer, h.bufKV.key, nil, argsNoValue)
```

Comment:
Shouldn't this be `appendArgBytes`? And then `h.trailer` should be reset before the `for` loop. And should there also be a `AddTrailerBytes`?

Reply:
Sure.
Having a `AddTrailerBytes` will be better.